config/sys.config

```diff
@@ -0,0 +1,17 @@
+[
```

Comment:
this has been a lib up until now and not a release, why does that need to change?